src/tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "compilerOptions": {
         "types": [],
         "baseUrl": "./",
-        "rootDir": "./"
+        "rootDir": "./",
+        "outDir": "../.temp/manualBuildCore",

```

Comment:
@sebavan : can this be a problem? (To define an outdir?)

Reply:
@sguillia Did you try to compile the project with `npm run build` to ensure your changes are not breaking the build?

Reply:
yup all good :-)